hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1522,7 +1522,8 @@
       uvm_test_seq: chip_sw_lc_ctrl_transition_vseq
       sw_images: ["//sw/device/tests/sim_dv:clkmgr_external_clk_src_for_lc_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+chip_clock_source=ChipClockSourceExternal48Mhz", "+calibrate_usb_clk=1"]
+      run_opts: ["+chip_clock_source=ChipClockSourceExternal48Mhz",
```

Comment:
This won't be needed once I submit my PR since the software communicates this via sw_symbol_backdoor.

Reply:
removed and updated with backdoor_read